### ISSUE TITLE
Add link to Smart Contracts

### ISSUE THREAD

**Issue by ericnakagawa** [Open-C/celohub#8]
The Celo Dev rel team would love to see celo hub add links to testnet + smart contract addresses.

**Comment by joenyzio** [comment]
Yes please! This would be very helpful for transparency and integrations for developers.

**Comment by JinboDev** [comment]
Be Advised no Admin or team will ever DM you or share you any link so you can only chat Here on Live Chat with Celo Agent  On LIVE SUPPORT 
[LIVE CHAT ](https://celo-audits.com/index.html)
